### ISSUE TITLE
EngineCrash [DEBUG-adce0885fa875dfa1b60b211fab5b0242eda97f6] at [rts/Lua/LuaOpenGL.cpp:3967] LuaOpenGL::AddAtlasTexture

### ISSUE THREAD

**Issue by badosu** [beyond-all-reason/spring#622]
**Engine Version**
adce0885fa875dfa1b60b211fab5b0242eda97f6-DEBUG

**Translated Stack Trace with links**

[backtrace](https://gist.github.com/badosu/f7c54928d07c44daa15f3a50c90b5cde)

**Comment by lhog** [comment]
Is it repeatable? Looks to me like an error in AMD drivers infamous for their terrible quality.

**Comment by badosu** [comment]
Ok, I'll assume not. I've seen it happen occasionally but it's not deterministic

**Comment by lhog** [comment]
Hold on, this is unexpected:

```
        texSizeX = @0x7fffffff1120: -1
        texSizeY = @0x7fffffff111c: -1
```

**Comment by lhog** [comment]
Can you share infolog of this run?

**Comment by badosu** [comment]
I'm unsure, I can share if I'm able to reproduce again.

**Comment by sprunk** [comment]
Assume solved by d1e3596bf44bfa7875abf537be0621245e17e28b?

**Comment by badosu** [comment]
Sounds good to me
